grafana-plugin/src/containers/OutgoingWebhookForm/OutgoingWebhookForm.tsx

```diff
@@ -370,46 +275,110 @@ export const OutgoingWebhookForm = observer((props: OutgoingWebhookFormProps) =>
                 </Button>
               )}
               <WithPermissionControlTooltip userAction={UserActions.OutgoingWebhooksWrite}>
-                <Button form={form.name} type="submit" disabled={data.is_legacy}>
-                  {isNewOrCopy ? 'Create' : 'Update'}
+                <Button type="submit" onClick={handleSubmit(onSubmit)} disabled={data.is_legacy}>
+                  Create
                 </Button>
               </WithPermissionControlTooltip>
             </HorizontalGroup>
           </div>
         </div>
-      </>
-    );
-  }
-});
+      </div>
+    </Drawer>
+  );
+};
+
+interface EditWebhookTabsProps {
+  id: OutgoingWebhookFormProps['id'];
+  data: any;
```

Comment:
We have couple of places in this component where we use `data: any`. I suggest to improve all those places. I guess we could use `ReturnType<typeof prepareDataForEdit>;` in each?